src/Sulu/Bundle/ContentBundle/Tests/app/AppKernel.php

```diff
@@ -21,5 +21,9 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         if (class_exists('Sulu\Bundle\SearchBundle\SuluSearchBundle')) {
             $loader->load(__DIR__ . '/config/search.yml');
         }
+
+        if ($this->environment === 'multiple_document_managers') {
```

Comment:
this is quite specific, should probably add a note saying what this is used for.